src/test/java/com/powsybl/openloadflow/ac/AcLoadFlowEurostagTutorialExample1Test.java

```diff
@@ -465,4 +465,26 @@ void maxOuterLoopIterationTest() {
         assertFalse(result.isFullyConverged());
         assertEquals(LoadFlowResult.ComponentResult.Status.MAX_ITERATION_REACHED, result.getComponentResults().get(0).getStatus());
     }
+
+    @Test
+    void testWriteReadSlackBus() {
+        Network network = EurostagFactory.fix(EurostagTutorialExample1Factory.create());
+        network.getVariantManager().cloneVariant(network.getVariantManager().getWorkingVariantId(), "newVariant");
+        LoadFlowParameters parameters = new LoadFlowParameters();
+        parameters.setWriteSlackBus(true);
+        OpenLoadFlowParameters openLoadFlowParameters =
+                OpenLoadFlowParameters.create(parameters).setSlackBusSelectionMode(SlackBusSelectionMode.FIRST);
+        LoadFlowResult result = LoadFlow.run(network, parameters);
+        assertTrue(result.isFullyConverged());
+        SlackTerminal slackTerminal = network.getVoltageLevel("VLGEN").getExtension(SlackTerminal.class);
+        assertNotNull(slackTerminal);
+        assertNotNull(slackTerminal.getTerminal());
```

Comment:
you could also demonstrate, here, or at the end of the test, via assertions, that slackTerminal.getTerminal() is null in the `newVariant` variant, this would make the test purpose much more explicit.